src/error-code.ts

```diff
@@ -0,0 +1,23 @@
+export interface ErrorCode {
+  code: number;
+  messageText: string;
+}
+
+export namespace ErrorCodes {
+  /**
+   * The computations that [Amazon States Language](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-amazon-states-language.html)
+   * can do is restricted by JSON Path and the limited [Intrinsic Functions](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html). Currently, arithmetic expressions are not supported.
+   * ```ts
```

Comment:
I have been adding workarounds to the "Limitations" pages for event bus. Should errors/limitations include how (easy it is) to work around the limitation?

Reply:
Good point, we should document workarounds for each of the error codes 👍